src/main/scala/za/co/absa/hyperdrive/trigger/persistance/DagInstanceRepository.scala

```diff
@@ -51,14 +55,22 @@ class DagInstanceRepositoryImpl extends DagInstanceRepository {
     } yield ()).transactionally
   ).map(_ => (): Unit)
 
-  override def getNewActiveDags(idToFilter: Seq[Long], size: Int): Future[Seq[DagInstance]] = db.run(
-    dagInstanceTable.filter(ji =>
-      !ji.id.inSet(idToFilter) && ji.status.inSet(DagInstanceStatuses.finalStatuses)
-    ).take(size).result
-  )
+  def getDagsToRun(idToFilter: Seq[Long], size: Int)(implicit executionContext: ExecutionContext): Future[Seq[DagInstance]] = {
```

Comment:
Please rename `idToFilter` to `runningIds` or something like this. Then it's clearer to follow the logic

Reply:
Done.